src/components/learner-credit-management/SubBudgetCard.jsx

```diff
@@ -106,9 +108,9 @@ const SubBudgetCard = ({
       isLoading={isLoading}
     >
       <Card.Body>
-        <Stack gap={4}>
+        <Stack>
```

Comment:
I would recommend leaving the `Stack` component here for the spacing between the `Card.Header` and the below `Card.Section`, but only provide the `gap={4.5}` prop conditionally depending on the budget status. Also note that this `gap` value should be `4.5` to be consistent with the `Card.Header` default top margin.